packages/compat/uni-builder/src/shared/defaults.ts

```diff
@@ -0,0 +1,120 @@
+import { UniBuilderRspackConfig } from '../types';
+
+export const DEFAULT_PORT = 8080;
+export const DEFAULT_DATA_URL_SIZE = 10000;
+export const DEFAULT_MOUNT_ID = 'root';
+export const DEFAULT_ASSET_PREFIX = '/';
+export const DEFAULT_DEV_HOST = '0.0.0.0';
+
+export const getDefaultDevConfig = () => ({
+  hmr: true,
+  https: false,
+  port: DEFAULT_PORT,
+  assetPrefix: DEFAULT_ASSET_PREFIX,
+  startUrl: false,
+  progressBar: true,
+  host: DEFAULT_DEV_HOST,
```

Comment:
Why not just override/add some default config

Reply:
That' true. 

But there are some other considerations:

- How can we know the difference of default config between rsbuild and uni-builder? If we maintain the diff, it's easily to see the difference. If we maintain two separate default config, this will be harder.
- When Rsbuild introduced a new config, should we add the default value to uni-builder?